public_suffix_list.dat

```diff
@@ -14853,6 +14853,11 @@ cloudsite.builders
 myradweb.net
 servername.us
 
+// Raidboxes GmbH : https://raidboxes.de
+// Submitted by Auke Tembrink <auke@raidboxes.de>
```

Comment:
Please use a non-personal address if possible.

Reply:
Thanks for the feedback. I have changed it to the team address.